app/electron/main.js

```diff
@@ -137,12 +174,36 @@ async function createWindow() {
       if (allowedPermissions.includes(permission)) {
         permCallback(true); // Approve permission request
       } else {
-        /* !TODO LOG
-        console.error(
-          `The application tried to request permission for '${permission}'.
-          This permission was not whitelisted and has been blocked.`
-        );
-        */
+        log.catchErrors({
+          showDialog: false,
+          onError(error, versions, submitIssue) {
+            dialog
+              .showMessageBox({
+                title: "An error occurred",
+                message:
+                  `The application tried to request permission for '${permission}' This permission was not whitelisted and has been blocked.`,
+                detail: error.stack,
+                type: "error",
+                buttons: ["Ignore", "Report", "Exit"],
+              })
+              .then((result) => {
+                if (result.response === 1) {
+                  submitIssue("https://github.com/djunicode/ergo/issues/new", {
+                    title: `Permission issue for ${versions.app}`,
+                    body:
+                      `Error:\n\`\`\`${error.stack}\n\`\`\`\n` +
+                      `OS: ${versions.os}`,
+                  });
+                  return;
+                }
```

Comment:
Are you sure this is how error handling is to be set up ? @namandangi  can you once verify this? The impression that I am getting from electron-log is that this should be a global config and somehow it will handle uncaught errors, but not sure : check https://github.com/megahertz/electron-log/blob/master/docs/catch.md